hack/.import-aliases

```diff
@@ -0,0 +1,8 @@
+{
+  "k8s.io/api/apps/v1": "appsv1",
+  "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1": "mcsv1alpha1",
+  "github.com/karmada-io/karmada/pkg/apis/work/v1alpha1": "workv1alpha1",
+  "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1": "policyv1alpha1",
+  "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1": "clusterv1alpha1",
+  "github.com/karmada-io/karmada/pkg/generated/clientset/versioned": "karmadaclientset"
```

Comment:
By the way, why do you think we should define aliases for the `generated` packages?